boards/arm/mimxrt1050_evk/Kconfig.defconfig

```diff
@@ -71,8 +71,18 @@ config ETH_MCUX_0
 
 endif # NETWORKING
 
+config DISPLAY_DRIVER
```

Comment:
Should this config not be defined in the CFB KConfig and better rename it to `CHARACTER_FRAMEBUFFER__DISPLAY_DEV_NAME`?
Further, remove `CHARACTER_FRAMEBUFFER_SHELL_DRIVER_NAME` and rework the samples and cf_shell.c to use this new config.

Reply:
ok